src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -161,7 +161,7 @@ private void ProcessType(INamedTypeSymbol typeSymbol)
 					builder.AppendLineInvariant("using AppKit;");
 					builder.AppendLineInvariant("#endif");
 
-					using (builder.BlockInvariant($"namespace {typeSymbol.ContainingNamespace}"))
+					using (typeSymbol.ContainingNamespace.IsGlobalNamespace ? DisposableAction.NoOp : builder.BlockInvariant($"namespace {typeSymbol.ContainingNamespace}"))
```

Comment:
```suggestion
					using (typeSymbol.ContainingNamespace.IsGlobalNamespace ? Uno.Actions.Null : builder.BlockInvariant($"namespace {typeSymbol.ContainingNamespace}"))
```

Reply:
`Uno.Actions.Null` is not `IDisposable` so I don't think it works in `using`.

Reply:
I can update the disposable action to `internal readonly static DisposableAction Null = new DisposableAction(Actions.Null);` though.

Reply:
Right I'm sorry, that's not the right one, My mistake :) There's this one:
https://github.com/unoplatform/uno/blob/c94d459528296a91d0ac6cac89b3c78304f11b49/src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions.Disposables/NullDisposable.cs#L24